### PR TITLE
Add code of conduct

### LINKED ISSUE #931: Add code of conduct
Add a code of conduct for Aesara. There is no current code of conduct. 

The [Django Project code of conduct](https://www.djangoproject.com/conduct/) seems reasonable.

### PR DESCRIPTION
**Pull request by dgerlanc**
Add an Aesara code of conduct adapted from the Django code of conduct.

Resolves #931.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  and we conduct ourselves professionally. Be kind to others. Do not insult or
+  put down other participants. Harassment and other exclusionary behavior aren't
+  acceptable. This includes, but is not limited to:
+
+    - Violent threats or language directed against another person.
+    - Discriminatory jokes and language.
+    - Posting sexually explicit or violent material.
+    - Posting (or threatening to post) other people's personally identifying
+      information ("doxing").
+    - Personal insults, especially those using racist or sexist terms.
```
Comment (brandonwillard) [inline CODE_OF_CONDUCT.md:58]:
```suggestion
    - Personal insults of any sort.
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  community should be respectful when dealing with other members as well as with
+  people outside the Aesara community.
+
+- __Be careful in the words that you choose.__ We are a community of professionals,
+  and we conduct ourselves professionally. Be kind to others. Do not insult or
+  put down other participants. Harassment and other exclusionary behavior aren't
+  acceptable. This includes, but is not limited to:
+
+    - Violent threats or language directed against another person.
+    - Discriminatory jokes and language.
```
Comment (brandonwillard) [inline CODE_OF_CONDUCT.md:54]:
```suggestion
    - Discriminatory jokes and language (e.g. the use of racist and sexist terms).
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    - Unwelcome sexual attention.
+    - Advocating for, or encouraging, any of the above behavior.
+    - Repeated harassment of others. In general, if someone asks you to stop,
+      then stop.
+
+- __When we disagree, try to understand why.__ Disagreements, both social and
+  technical, happen all the time and Aesara is no exception. It is important
+  that we resolve disagreements and differing views constructively. Remember
+  that we’re different. The strength of Aesara comes from its varied community,
+  people from a wide range of backgrounds. Different people have different
```
Comment (brandonwillard) [inline CODE_OF_CONDUCT.md:68]:
```suggestion
  that we’re different. Aesara benefits from its varied community and
  people from a wide range of backgrounds. Different people have different
```

**Review by ricardoV94 — COMMENTED** [review]
Looks good!

**Review by twiecki — DISMISSED** [review]

**Comment by dgerlanc** [comment]
Updated with @brandonwillard suggested changes.

**Review by brandonwillard — APPROVED** [review]

**Pull request merged**